packages/xo-server/src/xo-mixins/jobs/index.mjs

```diff
@@ -175,7 +175,7 @@ export default class Jobs {
           ? {
               mode: job.mode,
               reportWhen: job.settings['']?.reportWhen ?? 'failure',
-              shorterBackupReports: job.settings['']?.shorterBackupReports,
+              backupReportTpl: job.settings['']?.backupReportTpl ?? 'mjml',
```

Comment:
Avoid injecting a default value everywhere, try to keep it as limited as necessary.